### DISCUSSION TITLE
html5 compatibility

### DISCUSSION THREAD

**Discussion by RadicalRonik** [ikemen-engine/Ikemen-GO#2014] [Feature Request]
### Is your feature request related to a problem? Please describe.

Sites like Newgrounds doesn't support Ikemen GO, due to the presence of executable files, since .exe files could contain some virus.

### Describe the solution you'd like

Make Ikemen GO compatible with HTML5, for indie developers who want to show their games at sites like Newgrounds or Itch.io.

### Additional context

It would serve for indie developers to show their games before being downloaded.

**Comment by formula1** [comment]
Theres already discussions regarding porting to wasm here

https://github.com/ikemen-engine/Ikemen-GO/discussions/879

https://github.com/ikemen-engine/Ikemen-GO/discussions/1627

https://github.com/ikemen-engine/Ikemen-GO/discussions/901
